PROPOSALS.md

```diff
@@ -78,11 +78,30 @@ request feedback on feasibility before investing time and effort in the full pro
     * or just share patch files, in the same format as
       [`head.hackage`](https://gitlab.haskell.org/ghc/head.hackage) does.
 5. Implement the proposal.
-    * Note that you should follow [GHC's "Contributing a Patch" guidelines](https://gitlab.haskell.org/ghc/ghc/-/wikis/Contributing-a-Patch) when creating an MR (e.g. [appropriate commit messages](https://gitlab.haskell.org/ghc/ghc/-/wikis/Contributing-a-Patch#22-commit-messages)),
+    * Register at https://gitlab.haskell.org/ghc/ghc.
+    * If your account is not approved soon (this is a manual process to combat spam),
+      ask for approval in your CLC discussion thread, someone will take care of it.
+    * Fork GHC repository and implement your proposal in a non-`master` branch.
+      (It's probably best if you create a new branch rather than working `master`, see below.)
```

Comment:
Sorry, there was a typo in my original suggestion:

```suggestion
      (It's probably best if you create a new branch rather than working in `master`, see below.)
```

Reply:
(Maybe explaining my line of thinking will help: I don't see any value in asking the implementer to use a non-`master` branch at this stage. It seems like it will just add to their burden. Creating a new branch specifically for the merge request is easy, so I suggest deferring that instruction until later (as we already have done))